crates/core/src/globals.rs

```diff
@@ -70,15 +70,18 @@ where
     T: Write + 'static,
 {
     move |ctx: &Context, _this: &Value, args: &[Value]| {
+        // write full string to in-memory destination before writing to stream since each write call to the stream
+        // will invoke a hostcall
```

Comment:
```suggestion
    // Write full string to in-memory destination before writing to stream since each write call to the stream
    // will invoke a hostcall.
    ```